### PR TITLE
chore: add Relative path of root README.md

### PR DESCRIPTION
**Pull request by 15IITian**
Added the relative path of root README.md file to those Crate's Cargo.toml which do not have its own README
#3924

### CODE REVIEW

File: fedimint-wasm-tests/Cargo.toml [hunk 1]
```diff
@@ -3,6 +3,7 @@ name = "fedimint-wasm-tests"
 version = "0.0.0"
 edition = "2021"
 license = "MIT"
+readme = "../README.md"
 publish = false
```
Comment (elsirion) [inline fedimint-wasm-tests/Cargo.toml:7]: Not really needed for `Cargo.toml`s with `publish=false` since they aren't published to crates.io

**Review by elsirion — COMMENTED** [review]
The meta data isn't really needed for unpublished crates, so I think we should just not touch their `Cargo.toml`s. Otherwise looks good :)

File: fedimint-wasm-tests/Cargo.toml (hunk 1 above)
Comment (15IITian) [reply fedimint-wasm-tests/Cargo.toml:7]: OK , I'll remove them from such Cargo.toml.

**Review by elsirion — APPROVED** [review]
nit: normally you don't undo changes from earlier commits in later ones, but would rather [edit old commits](https://www.atlassian.com/git/tutorials/rewriting-history) and force-push.

Feel free to fix it, I'll re-ACK, but it's not a huge deal here since the PR is very small.

**Comment by Maan2003** [comment]
needs rebase

**Comment by 15IITian** [comment]
> needs rebase
I am not getting -> please guide me to this problem.

**Comment by 15IITian** [comment]
> needs rebase
I am not getting -> please guide me to this problem.

**Comment by 15IITian** [comment]
> needs rebase

I am not getting how to solve this problem -> kindly please help me out .

**Comment by 15IITian** [comment]
>  Idk how you even got into this state.


It was my fault as after successfully editing  the older commits (to solve that unpublished crate issue which was merged ) in this branch  by  `interactive rebase `-> By mistake , I rebase this branch (not the master branch)  and then push all the work .
Therefore , it is showing all the commits which are just after that commit .

**Comment by 15IITian** [comment]
> Idk how you even got into this state, but copying the commit hashes of the commits you actually want, a hard reset on an up-to-date master and then cherry-picking is probably the fastest way to fix it. You can find some documentation on how to use git here https://git-scm.com/
> 
> Screenshot 2024-01-17 at 16-57-36 chore add Relative path of root README md by 15IITian · Pull Request #4038 · fedimint_fedimint

I 'll try it

**Comment by 15IITian** [comment]
> Idk how you even got into this state

I got the problem:
I was trying to edit merged PR' commit which causes to reopen rest other merged PR's as it changes its commit hash :
Screenshot from 2024-01-18 09-48-38

**Comment by elsirion** [comment]
The audit CI error isn't your fault.

**Review by dpc — APPROVED** [review]

**Pull request merged**